ChibiOS/BrainPad2/board.h

```diff
@@ -35,7 +37,7 @@
  * Board identifier.
  */
 #define BOARD_BRAINPAD2
-#define BOARD_NAME                  "GHI BOARD_BRAINPAD2"
+#define BOARD_NAME                  "GHI_BOARD_BRAINPAD2"
```

Comment:
By looking at the [Brainpad site](https://www.brainpad.com/pulse/) this target probably should be called BRAINPAD_PULSE...

Reply:
That is the new sitcore version. Going by mouser it was BRAINPAD-BP2

Reply:
Got you. But according to Mouser that's a 40x model. And zooming in to the board pic:
![image](https://user-images.githubusercontent.com/1881520/146023661-fe260769-d369-4c48-93f2-adbd7b6ec000.png)

Reply:
Exactly, and that is what I changed it to...

Reply:
🤪 you're correct of course... I was looking it the other way around... my bad!